extension/src/init.js

```diff
@@ -16,13 +16,17 @@ if (envConfig.keepAliveTimeout !== undefined)
   server.keepAliveTimeout = envConfig.keepAliveTimeout
 server.listen(port, async () => {
   const ctpProjectKeys = config.getAllCtpProjectKeys()
-  await pMap(ctpProjectKeys, async (ctpProjectKey) => {
-    const ctpConfig = config.getCtpConfig(ctpProjectKey)
-    if (ctpConfig.ensureResources) {
-      const ctpClient = ctpClientBuilder.get(ctpProjectKey)
-      await ensureResources(ctpClient)
-    }
-  })
+  await pMap(
+    ctpProjectKeys,
+    async (ctpProjectKey) => {
+      const ctpConfig = config.getCtpConfig(ctpProjectKey)
+      if (ctpConfig.ensureResources) {
+        const ctpClient = ctpClientBuilder.get(ctpConfig)
+        await ensureResources(ctpClient)
+      }
+    },
+    { concurrency: 5 }
```

Comment:
What is the benefit of limiting concurrency here?

Reply:
Benefit is in case extension has hundreds of projects at once, then this module will call only 5 projects at once to ensure resources.

Reply:
It could stay but imo requests are not going to the same server or same ctp client, so putting a concurrency for it does not make sense in terms of the node's working model. 

Also, do you know what could happen if ensure resources fails for a project but then stops? In the next run, it should not create resources on those projects, right?

Reply:
I removed pMap as it's not very useful for our usage anyway: https://github.com/commercetools/commercetools-adyen-integration/pull/541/commits/a2fda293e4e3dbf23d0b28ec262f3bea95a07d69

If it fails for a project, it should fail the whole extension. During the next run, it first checks if resources are there. If not, it will create, if resources are there, it will not create again.